moodlecli/main.py

```diff
@@ -249,11 +249,13 @@ def export_users(source_course_id, bucket_name, key):
     bucket"""
```

Comment:
General PR comment: Can we update the title per our conventions?

Reply:
😳

Reply:
@mjoss17 - Nudging this comment as the PR title is still the auto-generated string.